src/docs/patterns/account-details.mdx

```diff
@@ -0,0 +1,140 @@
+            { title: 'Account Settings Page', hash: '#account-settings-page' },
+            { title: 'In the Navigation Drawer', hash: '#in-the-navigation-drawer' },
+            { title: 'Via User Menu', hash: '#via-user-menu' },
+            { title: 'Contents', hash: '#contents' },
+            { title: 'Design Specifications', hash: '#design-specifications' },
+            { title: 'Developers', hash: '#developers' },
+        ]}
+    />
+    <Divider />
+</React.Fragment>
+
+# Account Settings Page
+
+<ImageGrid
+    images={[AccountSettingsPage]}
+    caption={`A basic account settings page with typical user information and options.`}
+/>
+
+At a minimum, your application should implement a dedicated account settings page for viewing and editing a user's account details, preferences, settings, etc. This page should also include a way for users to take account related actions. The style of this page should be consistent with other settings pages in your application, if applicable.
+
+<React.Fragment>
+    <Divider />
+</React.Fragment>
+
+# In the Navigation Drawer
+
+<ImageGrid
+    images={[DrawerAccountDetails]}
+    caption={`A navigation drawer showing basic user details and account-related actions.`}
+/>
+
+If your application is using [Drawer Navigation](/patterns/navigation#drawer-navigation), you can include quick-access user information and actions in the Drawer. Commonly, this involves placing an an avatar in the drawer header along with basic information about the user, such as their name and username / email address. Additional account actions and a link to the account settings page can be placed in the drawer body just above the footer (see the [contents](/patterns/account-details#contents) section below).
+
+This approach is most effective when there are relatively few account-related actions. You should **NOT** use this approach if you are using a navigation rail or a primary navigation mechanism other than a drawer.
+
+<React.Fragment>
+    <Divider />
+</React.Fragment>
+
+# Via User Menu
+
+<ImageGrid
+    images={[DesktopUserMenu]}
+    regularWidth
+    caption={`The user menu is controlled by an Avatar button and displays useful user-related information and actions.`}
+/>
+
+Another way to present user information and actions is with the user menu. The user menu appears as an Avatar in the [App Bar](/patterns/appbar), usually displaying a user-selected profile picture. If your application does not support user-elected images, you can default to displaying the user's initials or a generic [person icon](/style/iconography?icon=Person&isMaterial=true).When clicked, the avatar opens a menu with the user-related content, such as links to the user profile / settings, log out functionality, etc. (see the [contents](/patterns/account-details#contents) section below). Further, the menu should also display basic information about the user when opened, such as their name and / or username.
```

Comment:
generic [person icon].When

Needs a space after the period; looking great so far!